### ISSUE TITLE
[Feature Request] Disable Stair / Wall / Slab etc. generation only for vanilla items / blocks or change output

### ISSUE THREAD

**Issue by D3nnis3n** [MinecraftModDevelopmentMods/BaseMetals#187]
Hey,
i would like to be able to disable the wall, slab, stair etc. gewneration for vanilla items and blocks or at least be able to Change the amount of Output. I use "Vanilla Building Tools" for e.g. Diamond and emerald stairs, but it also adds all the other Things of vanilla, not only metals, but unfortunatley it only gives 3 walls not 6 like BaseMetals does. I might be able to fix that with the Tweaker mods, but those are incompatible with the recent malisis core build :/

**Comment by dshadowwolf** [comment]
okay - I'll have to run this past @jriwanek to see how to handle it. Check the Base Metals config - there is a setting for disabling just about everything. And... let me poke Jared about the Malisis non-compat.

**Comment by D3nnis3n** [comment]
https://github.com/Ordinastie/MalisisCore/issues/117

The malisis Dev is already aware of this for quite some time. It will be fixed, but until then no tweakers.

I also wrote to the dev of vanilla building tools as tue output inconsistency comes from bis mod in the first place. You just do what Vanilla does. Unfortunately he has no config for the amount yet. I could disable stairs in Nase metals, nur this would disable it dir every metal, not just your Vanilla changes in these Blocks. I believe you are not the One who should Change anything here but as crafttweaker being incompatible with malisis for over a month i have no way to fix it independently from one of tue affected mods.

**Comment by dshadowwolf** [comment]
...ffs - even if done as a fooking passthrough...

Okay, that incompat is because Malisis is not implementing a function in "MalisisBlock" that is getting called indirectly via Vanilla and using it as an ItemBlock - where that function is expected (getSubBlocks)

Easy enough fix, too...

**Comment by D3nnis3n** [comment]
Well, it's his / hers decision to update whenever he likes, but as we make heavy use of door now we decided to move out crafttweaker until that as this can be reimplemented later. I'll just hope there will be an update soon. Nothing i could do until then. I'm happy for any developer that still has fun doing his Things and therefore have no Intention to complain. I just tried to politeley ask for a hotfix. If not - all good, too.
